README.md

```diff
@@ -73,28 +83,28 @@ the RFC has received ample discussion and enough of the tradeoffs have been
 discussed. The Shepherd Team will propose to either accept or reject the RFC
 after the FCP.
 
+##### RFC Categories
+In order to do do justice to the different aspects of documents that merit
+generation of broad community consensus via the RFC process, we cassify each
+RFC into _feature_, _process_ and _informational_ RFCs. All follow the same
+high-level process as described above, but each category requires a different
+"mode of discussion", templates, critarias and judgment that it is benefical
+to the overall RFC process to identify those categories explicitly.
+
 
 ## Process from Creation to Merge
 
-*In short, to get a major change included in Nix or Nixpkgs, one must
+*In short, to get a major change included in Nix, Nixpkgs or the Ecosystem, one must
```

Comment:
```suggestion
*In short, to get a major change included in Nix, Nixpkgs or the ecosystem, one must
```
maybe? I'm not a native speaker. Also I don't know if I even should propose typo fixes or if you wanted to read over it anyways.